server/routes/api/landing_page.py

```diff
@@ -75,23 +75,34 @@ def fill_translation(chart):
 
 
 # TODO: add test for chart_config for assumption that each combination of stat vars will only have one config in chart_config.
-def build_spec(chart_config):
+def build_spec(chart_config, i18n=True):
```

Comment:
could you add a comment about the param (and that it should only be used for testing?)

Reply:
Done